tests/spec/core/id-headers-spec.js

```diff
@@ -46,7 +46,7 @@ describe("Core - ID headers", () => {
       expect(intro.getAttribute("aria-label")).toBe("Permalink for Section");
       const custom = doc.querySelector("#custom-id a.self-link");
       expect(custom.getAttribute("aria-label")).toBe(
-        "Permalink for Section 2."
+        "Permalink for Section 3."
```

Comment:
What happened here?

Reply:
This is correct... removing the `<p>BLAH</p>` above (which we won't support), causes the sections to be ordered correctly... so this is now "3. Pass":

<img width="284" alt="Screen Shot 2022-01-31 at 12 06 58 pm" src="https://user-images.githubusercontent.com/870154/151726049-81f2657d-b96d-465e-b583-dc262fbed01f.png">